cmd/di.go

```diff
@@ -20,12 +20,12 @@ package cmd
 import (
 	"time"
 
-	"github.com/asaskevich/EventBus"
 	log "github.com/cihub/seelog"
 	"github.com/ethereum/go-ethereum/accounts/keystore"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/mysteriumnetwork/node/blockchain"
+	"github.com/mysteriumnetwork/node/bus"
```

Comment:
I'm not sure, but just `bus` sounds not clear. Maybe it's better to name package `eventBus`? 
```suggestion
	"github.com/mysteriumnetwork/node/eventBus"
```